platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/merkle/map/internal/ExpectedFCMFamilyImpl.java

```diff
@@ -344,7 +344,7 @@ public long getNextIdToCreate() {
      * 		if it is true, return a MapKey in corresponding selfEntitiesList
      * @param restrictedCount
      * 		if non-zero, this will only select a subset of this many items out of the appropriate entity list
-     * 		convering the range.  So if there were 9 entities in the list and this was 2, you'd get either 0 or 8
+     * 		converting the range.  So if there were 9 entities in the list and this was 2, you'd get either 0 or 8
```

Comment:
I think this (and the other instance) should be `covering` (without the 'n' and the 't')
```suggestion
     * 		covering the range.  So if there were 9 entities in the list and this was 2, you'd get either 0 or 8
```

Reply:
Fixed as suggested